readme.md

```diff
@@ -137,11 +139,11 @@ Validate user's input against the entire rule set of your `Survey` using Laravel
 ```php
 class SurveyEntriesController extends Controller
 {
-    public function store(Survey $survey, Request $request)
+    public function store(Request $request, Survey $survey)
     {
-        $answers = $request->validate($request, $survey->rules);
+        $answers = $request->validate($survey->rules, $request->all());
```

Comment:
I think we can drop the second argument. 

```suggestion
        $answers = $request->validate($survey->rules);
```

Reply:
Thanks for pointing out. I confirmed that it's working without the 2nd param.